src/debug/inc/arm/primitives.h

```diff
@@ -30,7 +30,11 @@ typedef DPTR(CORDB_ADDRESS_TYPE)    PTR_CORDB_ADDRESS_TYPE;
 #define STACKWALK_CONTROLPC_ADJUST_OFFSET 2
 
 #define CORDbg_BREAK_INSTRUCTION_SIZE 2
+#ifdef LINUX32
```

Comment:
Ok, the break instruction code makes sense to be Linux specific then. But is the instruction address reported by the SIGTRAP on Unix ARM32 also expected to differ per Unix platform? That was actually the part that I had on my mind when I have responded to your PR.